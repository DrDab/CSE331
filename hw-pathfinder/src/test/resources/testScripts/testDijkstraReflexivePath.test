# Test if Dijkstra produces correct output on graph with reflexive path

# Create graph g
CreateGraph g

# Add city names to graph g
AddNode g LosSantos

# Test pathfinding
FindPath g LosSantos LosSantos
