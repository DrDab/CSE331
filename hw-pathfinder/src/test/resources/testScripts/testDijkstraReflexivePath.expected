# Test if Dijkstra produces correct output on graph with reflexive path

# Create graph g
created graph g

# Add city names to graph g
added node LosSantos to g

# Test pathfinding
path from LosSantos to LosSantos:
total cost: 0.000
